src/OrchardCore.Modules/OrchardCore.Contents/AdminMenu.cs

```diff
@@ -55,6 +63,12 @@ public async Task BuildNavigationAsync(string name, NavigationBuilder builder)
                         var ci = await _contentManager.NewAsync(contentTypeDefinition.Name);
                         var cim = await _contentManager.PopulateAspectAsync<ContentItemMetadata>(ci);
                         var createRouteValues = cim.CreateRouteValues;
+                        createRouteValues.Add("returnurl", _linkGenerator.GetPathByRouteValues(_httpContextAccessor.HttpContext, "", new
```

Comment:
It probably matches case insensitive, but shouldn't it be `returnUrl` ?

Reply:
Why we introduce a static field for it, because it has been used in many places and to avoid case comparison

Reply:
In fact if we look at the login page it's `ReturnUrl` but here I kept it lowercase. Maybe a task would be to make them all consistent throughout the app. Though, I'm pretty sure it will work any case.

Reply:
RouteValueDictionary is case insensitive so anyway it works

Personally, for lisibility i prefer camel case `returnUrl` as it is done in some places as `action`, `controller`, `contentItemId`, we also have pascal case in some other places as `ContentItemId`.

Reply:
Yeah I made it consistent for the url as lowercase for cross-plat compatibility. The vars can remain camel cased.